pyoverkiz/utils.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import re
+
+from pyoverkiz.const import LOCAL_API_PATH
+from pyoverkiz.models import OverkizServer
+
+
+def generate_local_server(
+    host: str,
+    name: str = "Somfy Developer Mode",
+    manufacturer: str = "Somfy",
+    configuration_url: str | None = None,
+) -> OverkizServer:
+    """Generate OverkizServer class for connection with a local API (Somfy Developer mode)."""
+    return OverkizServer(
+        name=name,
+        endpoint=f"https://{host}{LOCAL_API_PATH}",
+        manufacturer=manufacturer,
+        configuration_url=configuration_url,
+    )
+
+
+def is_overkiz_gateway(gateway_id: str) -> bool:
```

Comment:
What’s an additional gateway?

Reply:
Some users have a thermostat + protect system, which is also listed as a gateway. We have a JSON definition of one of them in our fixtures folder.